build-test.cmd

```diff
@@ -253,6 +306,17 @@ echo. -? -h -help: view this message.
 echo Build architecture: -buildArch: only x64 is currently allowed ^(default: x64^).
 echo Build type: -buildType: one of Debug, Checked, Release ^(default: Debug^).
 echo updateinvalidpackageversions: Runs the target to update package versions.
+echo buildagainstpackages: builds tests against restored packages, instead of against a built product.
+echo runtimeid ^<ID^>: Builds a test overlay for the specified OS (Only supported when building against packages). Supported IDs are:
```

Comment:
This will be a hard list to maintain as RIDs increase/change. I would suggest you point to Microsoft.NETCore.Platforms's runtime.json file int he CoreFX repo instead.